package.json

```diff
@@ -28,52 +28,52 @@
   },
   "dependencies": {
     "@auth0/nextjs-auth0": "^1.7.0",
-    "@fortawesome/fontawesome-svg-core": "^1.2.36",
-    "@fortawesome/free-brands-svg-icons": "^5.15.4",
-    "@fortawesome/free-solid-svg-icons": "^5.15.4",
+    "@fortawesome/fontawesome-svg-core": "^1.3.0",
+    "@fortawesome/free-brands-svg-icons": "^6.0.0",
+    "@fortawesome/free-solid-svg-icons": "^6.0.0",
     "@fortawesome/react-fontawesome": "^0.1.17",
-    "@prisma/client": "^3.9.1",
-    "@sentry/nextjs": "^6.17.4",
+    "@prisma/client": "^3.9.2",
+    "@sentry/nextjs": "^6.17.9",
     "date-fns": "^2.28.0",
     "dayjs": "^1.10.7",
     "gravatar-url": "^4.0.1",
-    "mobx": "^6.3.13",
+    "mobx": "^6.4.0",
     "mobx-devtools-mst": "^0.9.30",
-    "mobx-react": "^7.2.1",
-    "mobx-state-tree": "^5.1.1",
-    "next": "^12.0.10",
+    "mobx-react": "^7.3.0",
+    "mobx-state-tree": "^5.1.3",
+    "next": "^12.1.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-linkify": "^1.0.0-alpha",
     "react-redux": "^7.2.6",
     "react-tooltip": "^4.2.21",
-    "sass": "^1.49.7",
+    "sass": "^1.49.8",
     "scss": "^0.2.4",
     "sentry": "^0.1.2",
     "styled-components": "^5.3.3"
   },
   "devDependencies": {
     "@kachkaev/eslint-config-react": "^0.4.5",
     "@kachkaev/markdownlint-config": "^0.3.0",
-    "@netlify/plugin-nextjs": "^4.2.4",
-    "@next/eslint-plugin-next": "^12.0.10",
-    "@types/node": "^16.11.22",
+    "@netlify/plugin-nextjs": "^4.2.7",
+    "@next/eslint-plugin-next": "^12.1.0",
+    "@types/node": "^17.0.18",
```

Comment:
We don’t use node directly, so the difference between 16 and 17 should not change anything for us.